Assets/MRTK/Core/Inspectors/Profiles/MixedRealitySceneSystemProfileInspector.cs

```diff
@@ -19,11 +19,14 @@ public class MixedRealitySceneSystemProfileInspector : BaseMixedRealityToolkitCo
         const float DragAreaOffset = 10;
         const float LightingSceneTypesLabelWidth = 45;
 
+        private static string defaultSceneContent =
```

Comment:
Could this be `const`?

Reply:
Sure

Reply:
was matching the existing private static string variables... i am happy to change all of them